src/stores/snapshot-slice.ts

```diff
@@ -319,7 +319,7 @@ window.ipcListener.subscribe( 'user-data-updated', ( _, payload ) => {
 	}
 } );
 
-function getCreateProgress( action: PreviewCommandLoggerAction ): [ string, number ] {
+function getOperationProgress( action: PreviewCommandLoggerAction ): [ string, number ] {
```

Comment:
This is unrelated to the `.editorconfig` file, I just realized that this function should no longer allude to the `preview create` command specifically, so I took this opportunity to rename it. The change is so small I did not think it deserved a separate PR.